hadoop-project/pom.xml

```diff
@@ -1076,6 +1075,9 @@
         <artifactId>jetty-servlet-tester</artifactId>
         <version>${jetty.version}</version>
       </dependency>
+      <!-- Hadoop's own codebase is not supposed to use commons-logging but some of the
```

Comment:
Do we really need to declare this? If other dependencies depend on it, maven will pull it in?